src/core-plugins/CoreCommands.ts

```diff
@@ -83,7 +86,7 @@ export default class CoreCommands extends Plugin {
                 MessageActions.sendMessage(ctx.channel.id, {
                     content: `**Debug Info:**
                         > Discord: ${DebugInfo.discordVersion}
-                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins)
+                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins) Custom Bundle: ${customBundle}
```

Comment:
I feel like this should be on a new line just to keep everything consistent

Reply:
done